src/components/DefineCompany/YourCompany.vue

```diff
@@ -0,0 +1,247 @@
+<template>
```

Comment:
This is similar to SummaryDefineCompany (which I did not delete) but this is laid out differently to handle Correct/Undo and slightly different sub-components.